game.cpp

```diff
@@ -89,23 +91,30 @@ void start_menu(main_character &character) {
   printf("%s", INTRODUCTION.c_str());
   add_log(INTRODUCTION);
   int option;
+  string message;
   while(1) {
     printf("%s", START_OPTIONS_MESSAGE.c_str());
-    scanf("%d", &option);
+    scanf(" %d", &option);
 
     switch(option) {
       case 1:
         select_difficulty_level();
         game_setup(character);
-        add_log("Uma nova partida foi iniciada!");
+        printf("Jogador digite seu primeiro nome:\n");
```

Comment:
"Jogador, digite o nome pelo qual deseja ser identificado:"
Nicknames são, hoje, algo intrínseco ao conceito de jogos

Reply:
Não acho que isso seja importante, eu dei enfoque no primeiro, porque se você for receber uma string e tiver um espaço no meio dá erro